.github/workflows/pr_metrics.yml

```diff
@@ -0,0 +1,59 @@
+name: Post-Sprint PR metrics
+on:
+  workflow_dispatch:
```

Comment:
You could change this to be a cron job every two weeks but if we ever moved the meeting wed have to update this.

Reply:
Yeah, i was probably going to set this up once I know it works the way its supposed to. OTOH, just manually generating it at the retro might be the most failsafe thing to do for the reason you mentioned.